### PR TITLE
ENH: adding attribute to link segmentAttributes sub-arrays to specific files

### PR DESCRIPTION
**Pull request by fedorov**
Compiles, need to test

Aims to address #253

### CODE REVIEW

File: include/dcmqi/itkTestMain.h [hunk 1]
```diff
@@ -89,7 +89,7 @@ int main(int ac, char *av[])
 {
   itk::FloatingPointExceptions::Enable();
 
-  double       intensityTolerance  = 2.0;
+  double       intensityTolerance  = 0.0001;
```
Comment (fedorov) [inline include/dcmqi/itkTestMain.h]: @che85 this was a nasty bug!

**Comment by fedorov** [comment]
@che85 if CIs are green and you have no edits requested, hold on merging for a little bit at least until we hear from @blezek/@potis.

File: include/dcmqi/itkTestMain.h (hunk 1 above)
Comment (che85) [inline include/dcmqi/itkTestMain.h]: That code came from BrainTools.... yeah it's a pretty high tolerance ...

Comment (fedorov) [inline include/dcmqi/itkTestMain.h]: Especially for comparing segmentations - they can be very different pixel-wise, but test would still succeed!

Comment (che85) [inline include/dcmqi/itkTestMain.h]: Glad that you noticed it....

**Comment by che85** [comment]
LGTM

**Comment by blezek** [comment]
@potis and I reviewed it today, looks good, but we don't have test data.  Could you suggest a test so we can verify that the Docker works as expected?

Go ahead and merge when you are ready.

**Comment by fedorov** [comment]
@blezek we have a test for this, see

https://github.com/QIICR/dcmqi/blob/master/apps/seg/Testing/CMakeLists.txt#L43 (itk2dcm)

and

https://github.com/QIICR/dcmqi/blob/master/apps/seg/Testing/CMakeLists.txt#L127 (dcm2itk for the output of the first one)

Is this what you were looking for?

**Pull request merged**